platform/ios/PolyPodApp/PolyPodTests/PolyOutTests.swift

```diff
@@ -146,4 +146,54 @@ class PolyOutTests: XCTestCase {
         
         wait(for: [expectation], timeout: 1.0)
     }
+    
+    func testImportOneArchive() {
+        let polyOut = PolyOut(session: NetworkSessionMock())
+        
+        let bundle = Bundle(for: type(of: self))
+        polyOut.activeFeature = Feature(path: URL(string: bundle.bundlePath)!, name: "Test", author: nil, description: nil, thumbnail: nil, thumbnailColor: nil, primaryColor: nil, links: nil)
+        
+        let url = bundle.url(forResource: "testZip", withExtension: "zip")!
+        
+        let expectation = XCTestExpectation(description: "Exp")
+        polyOut.importArchive(url: url.absoluteString, destUrl: nil) { newUrl in
+            XCTAssertTrue(newUrl != nil && newUrl != "", "newUrl is nil or empty")
```

Comment:
Nit: You should be consistent with how you check for emptiness, I see you did use 3 ways in this test function :). `isEmpty` should be preferred to `!= ""` or `.count > 0`.

Reply:
#diversity 😛